src/audio/CMakeLists.txt

```diff
@@ -166,7 +166,7 @@ set(mixer_sources ${mixer_src})
 set(src_sources src/src.c src/src_generic.c)
 set(asrc_sources asrc/asrc.c asrc/asrc_farrow.c asrc/asrc_farrow_generic.c)
 set(eq-fir_sources module_adapter/module_adapter.c module_adapter/module/generic.c eq_fir/eq_fir.c eq_fir/eq_fir_generic.c)
-set(eq-iir_sources eq_iir/eq_iir.c)
+set(eq-iir_sources module_adapter/module_adapter.c module_adapter/module/generic.c eq_iir/eq_iir.c)
```

Comment:
@singalsu pls see my plugin branch, I have a fix for this in testbench build. We can merge that incrementally though.

Reply:
Thanks, good to clean this up. I will do it after this PR for all module adapter components.